pkg/common/common_test.go

```diff
@@ -152,10 +152,13 @@ func TestStripDefaultPorts(t *testing.T) {
 }
 
 func TestGetImagePullPolicy(t *testing.T) {
+	pointerPullNever := corev1.PullNever
```

Comment:
nit: This isn't actually stored as a pointer yet so var name is a bit misleading.

Reply:
will try to think on other name

Reply:
fixed, found a better solution :raised_hands: